src/Uno.UI/UI/Xaml/Controls/Primitives/PickerFlyoutBase.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.UI.Xaml.Controls.Primitives
+{
+	public partial class PickerFlyoutBase : FlyoutBase
```

Comment:
Should have a protected ctor

Reply:
The _constructor_ should be protected, the class itself must be public.

Reply:
Like this?

Reply:
That's the one :)